other/s-z/update-image-tag/01-policy.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+apply:
+- update-image-tag.yaml
```

Comment:
Remember that this is a mutate existing policy and `spec.mutateExistingOnPolicyUpdate` here is set to `true`, which means we want to ensure that 1) the mutation occurs when the policy is created and 2) the mutation occurs when the policy is updated. Because you're creating the policy at the outside, you aren't testing for these scenarios.

Reply:
How do you want me to update the policy? I can check for the first case, but second?

Reply:
There's no update to the policy needed, there are test case updates needed.

Reply:
I've made the changes required.